src/Skia/Perspex.Skia.Desktop.TestApp/Program.cs

```diff
@@ -0,0 +1,85 @@
+﻿// Copyright (c) The Perspex Project. All rights reserved.
+// Licensed under the MIT license. See licence.md file in the project root for full license information.
+
+using System;
+using System.Linq;
+using System.IO;
+using System.Reactive.Linq;
+using Perspex;
+using Perspex.Animation;
+using Perspex.Collections;
+using Perspex.Controls;
+using Perspex.Controls.Html;
+using Perspex.Controls.Primitives;
+using Perspex.Controls.Shapes;
+using Perspex.Controls.Templates;
+using Perspex.Diagnostics;
+using Perspex.Layout;
+using Perspex.Media;
+using Perspex.Media.Imaging;
+using TestApplication;
+using Perspex.Rendering;
+#if PERSPEX_GTK
+using Perspex.Gtk;
+#endif
+using ReactiveUI;
+
+namespace Perspex.Skia.Desktop.TestApp
```

Comment:
We should probably remove these test apps as well if they're not working - when I ran this it just showed a red window.